chat-plugins/poll.js

```diff
@@ -218,7 +279,7 @@ exports.commands = {
 			room.poll.display();
 
 			this.roomlog(`${user.name} used ${message}`);
-			this.modlog('POLL');
+			this.modlog('POLL', null);
```

Comment:
:| no

Reply:
It gave me a typescript error when I didn't specify this.